test/cppsuite/test_harness/test.h

```diff
@@ -83,12 +83,12 @@ class test : public database_operation {
     thread_manager *get_thread_manager();
 
     protected:
+    const test_args &_args;
```

Comment:
Don't think we need this in the protected section and private is fine now, since we use the _args within the test class now.

Reply:
I think its fine to leave args here, someone will use it at some point and I'd rather not have to move it back again.

Reply:
I think we still need it in the protected section! So we can access it in the test